Sources/SwiftGenStrings/main.swift

```diff
@@ -0,0 +1,69 @@
+import ArgumentParser
+import Foundation
+import SwiftGenStringsCore
+
+extension URL: ExpressibleByArgument {
+
+    public init?(argument: String) {
```

Comment:
µ-nit no need for public

Reply:
Unfortunately yes, since it's a protocol requirement of `ExpressibleByArgument`